embedded-files/tekton/triggers.yaml

```diff
@@ -77,6 +77,14 @@ spec:
       - ref: staging-pipelinebinding
       template:
         ref: staging-triggertemplate
+      interceptors:
+        - cel:
```

Comment:
I think we should use a secret here instead of exposing the value? like explained here [0]

[0] https://tekton.dev/docs/triggers/eventlisteners/#github-interceptors

Reply:
Yes though your link is only for the github interceptor. For `cel` I think we need something like `compareSecret` from here: https://tekton.dev/docs/triggers/cel_expressions/

I didn't try that now because the password we use is already hardcoded here: https://github.com/SUSE/carrier/blob/main/internal/cli/clients/client.go#L1134 . 

A proper solution needs to handle secret generation/rotation and access (who can read it etc). This was a quick "fix" to make sure we don't forget to lock the triggering with a password. Right now, although we used a (hardcoded) secret when sending the request from Gitea, it was completely ignored on the Tekton side. I will create a new issue for the proper fix. Is that ok?

Reply:
Maybe I can create a test for this fix at least.

Reply:
Ok, No problem. I see a `TODO:` . That should do.

Reply:
Created a new issue: https://github.com/SUSE/carrier/issues/255